### ISSUE TITLE
Can't use where with multiple multiEntry columns

### ISSUE THREAD

**Issue by Jake-Thompson** [ujjwalguptaofficial/JsStore#319]
# Title 

Can't use where with multiple multiEntry columns

# Description 

query
```
connection.select({
    from: 'Items',
    order: {
        by: 'date_mod',
        type: 'desc'
    },
    where: {
        tags: 'has_tag',
        parents: 0
    }
})
```
current output
`[]`

expected output 
```
[{
    "id": 60,
    "parents": [ 0 ],
    "childs": [],
    "name": "this item has a tag has_tag",
    "desc": "",
    "done": false,
    "date_created": null,
    "date_mod": "2023-03-25T23:34:46.117Z",
    "tags": [ "has_tag" ],
    "ext": {}
}]
```

table definition 
```
{
    name: 'Items',
    columns: {
        id: {
            primaryKey: true,
            autoIncrement: true
        },
        parents: {
            dataType: 'array',
            default: [0],
            multiEntry:true
        },
        childs: {
            dataType: 'array',
            default: []
        },
        name: {
            notNull: true,
            dataType: 'string'
        },
        desc: {
            dataType: 'string',
            default: ''
        },
        done: {
            dataType: 'boolean',
            default: false
        },
        date_created: {
            dataType: 'date_time',
        },
        date_mod: {
            notNull: true,
            dataType: 'date_time'
        },
        tags: {
            dataType: 'array',
            default: [],
            multiEntry:true
        },
        ext: {
            dataType: 'object',
            default: {}
        }
    }
}
```

**Comment by Jake-Thompson** [comment]
I found a workaround which was suggested for when filtering on the same column https://github.com/ujjwalguptaofficial/JsStore/discussions/288

```connection.select({
        from: 'Items',
        order: {
            by: 'date_mod',
            type: 'desc'
        },
        where: [
            {tags: 'has_tag'},
            {parents: 0}
        ]
    })```
works, however it's not as intuitive

**Comment by ujjwalguptaofficial** [comment]
gotcha, will take a look.

**Comment by ujjwalguptaofficial** [comment]
have fixed in v - 4.5.4. Please test and let me know.
